blob/BreakingChanges.md

```diff
@@ -1,3 +1,7 @@
+Tracking Breaking Changes in 1.2.0
+
+* This module now supports Ruby versions to 2.4 through 2.7
```

Comment:
2.3 would be still in support matrix for latest Nokogiri.

Reply:
Added back support for 2.3 in e82f9e8.